hcert_spec.md

```diff
@@ -269,15 +269,6 @@ The Secretarial shall also:
 * Maintain a similar set of lists with 'test' certificates
 * Maintain a set of test certificates - at least one for each country.
 
-## Key Usage Policy Identifiers
-
-The document signing certificate MAY contain Extended key usage extension fields; these being:
-
-* OID 1.3.6.1.4.1.0.1847.2021.1.1        valid for test
-* OID 1.3.6.1.4.1.0.1847.2021.1.2        valid for vaccinations
-* OID 1.3.6.1.4.1.0.1847.2021.1.3        valid for recovery
-
-If none of these OIDs are present the document shall be considered valid for all three.
 
```

Comment:
As I mentioned in my general comment. I would hate us to completely loose the only containing mechanism we have. Even if we don't define any explicit identifiers here, we should state the general functions of extended key usage in DSC and how verifiers should treat them.

Reply:
@Razumain If we clearly define how the validators need to deal with them, I think it's good to have them (for the creators it has already been defined). Your modified text would exactly do so. My problem before was that it was not clear as a validator what should be done with them.
One minor issue: The validator would need to parse the X509 Certificate, which could be a bit cumbersome for certain platforms (not sure if there is a good X509 Parser on JS).

Reply:
I don't think the validators need to parse X.509 certificates at all. I foresee that there is an integration layer between the national DSC import from external countries where the certificates are presented to the validator with contextual information, e.g. mapped by kid and with attached metadata such as authorised HCERT types. The way this information is presented to the validator is a matter of local national preference. So at least in our case, all X.509 processing will be done in a single aggregation point in the country.

Reply:
@asitplus-pteufl  as to the validator parsing that - I would not expect so - as you probably have your National backend make consisce pre-parsed and short 'trust lists' with just the right info.

As to avoid having to do things like get X,Y out of certs in javascript on the client. As that gets very hairy.

Reply:
I realise after cleaning up the document and sorting it into the payload (DGC) and the container (hcert), that these policy identifiers are scoped to the DGC payload. We should point that out.

Reply:
I adopted the suggestion from @Razumain and added, that if such OIDs are present the verifiers SHALL check them. 
@Razumain what do you mean by these "extra documents"? How would that work for creators/verifiers?